text/0139-faster-erasure-coding.md

```diff
@@ -0,0 +1,237 @@
+# RFC-0139: Faster Erasure Coding
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Start Date**  | 7 March 2025                                                                    |
+| **Description** | Faster algorithm for Data Availability Layer                                                                    |
+| **Authors**     | ordian                                                                                            |
+
+## Summary
+
+This RFC proposes changes to the erasure coding algorithm and the method for computing the erasure root on Polkadot to improve performance of both processes.
+
+## Motivation
+
+The Data Availability (DA) Layer in Polkadot provides a foundation for
+shared security, enabling Approval Checkers and Collators to download
+Proofs-of-Validity (PoV) for security and liveness purposes respectively.
+As the number of parachains and PoV sizes increase, optimizing the performance
+of the DA layer becomes increasingly critical.
+
+[RFC-47](https://github.com/polkadot-fellows/RFCs/blob/main/text/0047-assignment-of-availability-chunks.md)
+proposed enabling systematic chunk recovery for Polkadot's DA to improve
+efficiency and reduce CPU overhead. However, systematic recovery assumes
+very good network connectivity to approximately one-third of validators (plus some
```

Comment:
a third of validators is required for chunk recovery also (the difference is that it can be any third). And it cannot be worse than regular recovery since the systematic recovery fall backs to regular recovery (keeping the systematic chunks it already got)